parameter/TypeElement.cpp

```diff
@@ -76,8 +76,9 @@ bool CTypeElement::hasMappingData() const
 // Element properties
 void CTypeElement::showProperties(std::string& strResult) const
 {
-    (void)strResult;
-    // Prevent base from being called in that context!
+    showDescriptionProperty(strResult);
+    // Prevent base from being called from the Parameter Type context.
```

Comment:
The name attribute is duplicated in the type and instance (common base Element). As a result we do not want to forwarded to the base class because it would duplicate the name attribute.